ui/apps/platform/cypress/helpers/ocpAuth.ts

```diff
@@ -0,0 +1,17 @@
+export function withOcpAuth() {
+    if (Cypress.env('OCP_BRIDGE_AUTH_DISABLED')) {
+        return;
+    }
+
+    cy.session('ocp-session-auth', () => {
+        cy.visit('/');
+        cy.url().should('contain', '/login?');
+        cy.get('input[name="username"]').type(Cypress.env('OPENSHIFT_CONSOLE_USERNAME'));
+        cy.get('input[name="password"]').type(Cypress.env('OPENSHIFT_CONSOLE_PASSWORD'));
+        cy.get('button[type="submit"]').click();
+        cy.url().should('contain', '/dashboards');
+        cy.contains('Skip tour', { timeout: 10000 }).click();
+        cy.wait(1000);
```

Comment:
Is there any alternative to `timeout` and `wait` or at least might it help to add 2 or 3 comments around those lines to describe the interaction?

Reply:
I removed both, and replaced this with a keypress of `Esc` once the page has fully loaded. If the tour modal is present, this closes it, otherwise it is a no-op.